packages/jest-environment-node/src/index.ts

```diff
@@ -88,7 +84,7 @@ class NodeEnvironment {
   }
 
   // Disabling rule as return type depends on script's return type.
-  runScript(script: Script): ?any {
+  runScript(script: Script): any | null | undefined {
```

Comment:
`null | undefined` is redundant since this is already any.

Reply:
It shouldn't be `any`. This is my current best guess at the type: https://github.com/facebook/jest/blob/419661f57ea8e3af26a788bb3db92127c854d13b/packages/jest-environment/src/index.ts#L30-L32

Reply:
Essentially, the return value this object: https://github.com/facebook/jest/blob/419661f57ea8e3af26a788bb3db92127c854d13b/packages/jest-transform/src/ScriptTransformer.ts#L559-L565
which is called like this: https://github.com/facebook/jest/blob/419661f57ea8e3af26a788bb3db92127c854d13b/packages/jest-runtime/src/index.ts#L689-L711

Reply:
> `null | undefined` is redundant since this is already any.

I agree, I was just trying to refactor without changing to much. I'll try to change it into a proper type instead of `any`.

Reply:
@SimenB WRT the `any` type here. I've just logged what is returned from runScript() and it looks like this (in  a separate app/jest setup):
```
{ 'Object.<anonymous>': [Function: Object.<anonymous>] }
```

JSON stringified it is an empty object:
```
'{}'
```

I'm not sure what to put as the return type here. I just don't have enough understanding of what is being done in the method. Just keep it as `any`? Or an unsealed object `{} | undefined | null`? Open to suggestions.

Reply:
Never mind, I'll follow your diff below.